src/contrib.jl

```diff
@@ -133,3 +134,62 @@ end
 
 # https://discourse.julialang.org/t/how-do-a-i-get-a-type-stripped-of-parameters/73465/11
 constructorof(::Type{T}) where T = Base.typename(T).wrapper
+
+
+# Used to remove Intervals.jl as a dependency
+# That is a heavy one, but having an Interval type is nice
```

Comment:
This comment might be better written as "Define our own minimal Interval type, inspired by Intervals.jl. We vendor it in to avoid adding the heavy Intervals.jl dependency."

That way it will still make sense some time from now when this specific removal has been forgotten. WDYT?

Reply:
Thanks, good suggestion.